assets/src/domain/blocks/event-attendees/types.ts

```diff
@@ -1,13 +1,16 @@
 import { BlockEditProps } from '@wordpress/blocks';
+import { RegistrationStatus } from '@dataServices/apollo/queries/registrations/types';
+import { EntityId } from '@dataServices/types';
+import { Order, AttendeesOrderByFields } from '@dataServices/apollo/queries';
 
 export interface EventAttendeesAttributes {
-	event: string; // GUID
-	datetime: string; // GUID
-	ticket: string; // GUID
-	status: string;
+	event: EntityId;
+	datetime: EntityId;
+	ticket: EntityId;
+	status: RegistrationStatus;
 	limit: number;
-	order: string;
-	orderBy: string;
+	order: Order;
+	orderBy: AttendeesOrderByFields | 'FIRST_THEN_LAST_NAME' | 'LAST_THEN_FIRST_NAME';
```

Comment:
just curious why those last two values were not part of `AttendeesOrderByFields`

Reply:
Because `AttendeesOrderByFields` is what our GQL supports i.e. `FIRST_NAME` and `LAST_NAME`, but for select control we gave those two extra options.